### PR TITLE
taxonomy: German ingredient

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -86318,7 +86318,7 @@ wikipedia:en:https://en.wikipedia.org/wiki/Xylose
 # Tonka beans are prohibited for use in food in the US: https://www.govinfo.gov/content/pkg/CFR-2006-title21-vol3/xml/CFR-2006-title21-vol3-sec189-130.xml
 en:tonka bean
 es:haba tonka
-de:Tonkabohn, Tonkabohnen
+de:Tonkabohn, Tonkabohnen, Tonkabohne
```
Comment (hangy) [inline taxonomies/ingredients.txt:86321]: I think "Tonkabohn" is probably a typo. If you search it on the internet, it only results in a handful of entries, which seems to confirm this. Let's just remove that alias, IMHO.
```suggestion
de:Tonkabohne, Tonkabohnen
```

Comment (moon-rabbitOFF) [reply taxonomies/ingredients.txt:86321]: You are right, I modified it, thanks

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -13698,6 +13698,11 @@ sl:modificiran koruzni škrob
 sr:modifikovani kukuruzni skrob
 sv:modifierad majsstärkelse
 
+<en:modified starch
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:13701]: I think we could add "modified" and "partially modified" in ingredients_processing.txt, that way we could (eventually) remove all the related entries in ingredients.txt.

But let's merge this PR first, we can do it later.

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**